aimmo-game/simulation/avatar/avatar_wrapper.py

```diff
@@ -38,7 +30,8 @@ def _construct_action(self, data):
         action_data = data['action']
         action_type = action_data['action_type']
         action_args = action_data.get('options', {})
-        action_args['avatar'] = self
+        action_args['avatar'] = self.user_id
+        action_args['origin'] = self.location
```

Comment:
Non-obvious place to update due to [`origin` name change](https://github.com/ocadotechnology/aimmo/pull/153#discussion_r76992744).